### PR TITLE
docs: "What's new in Svelte" November newsletter

### PR DESCRIPTION
**Pull request by DreaminDani**

### CODE REVIEW

File: apps/svelte.dev/content/blog/2024-12-01-whats-new-in-svelte-december-2024.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+authorURL: https://dreamindani.com
+---
+
+I can't believe it's already the last newsletter of the year! So much has happened in the Svelte community - from two great Svelte Summits to the release Svelte 5 🎉
+
+As we look forward to the next year, let's dive into what's new in the past month...
+
+## What's new in Svelte
+
+The bugfix versions in 5.1.x addressed a bunch of issues reported by the community. So if you're noticing anything weird, make sure you upgrade to 5.1.17 or later and, as always, check out the [CHANGELOG](https://github.com/sveltejs/svelte/blob/main/packages/svelte/CHANGELOG.md) for all the bug fixes.
```
Comment (benmccann) [inline apps/svelte.dev/content/blog/2024-12-01-whats-new-in-svelte-december-2024.md:14]:
```suggestion
The bugfix versions in 5.1.x and 5.2.x addressed a bunch of issues reported by the community. So if you're noticing anything weird, make sure you upgrade to the latest version and, as always, check out the [CHANGELOG](https://github.com/sveltejs/svelte/blob/main/packages/svelte/CHANGELOG.md) for all the bug fixes.
```

File: apps/svelte.dev/content/blog/2024-12-01-whats-new-in-svelte-december-2024.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+The features since last month can be found below:
+
+- Static attributes are now inlined better in the compiled output (**5.2.0**, [#14269](https://github.com/sveltejs/svelte/pull/14269))
```
Comment (benmccann) [inline apps/svelte.dev/content/blog/2024-12-01-whats-new-in-svelte-december-2024.md:18]: This was reverted
```suggestion
```

File: apps/svelte.dev/content/blog/2024-12-01-whats-new-in-svelte-december-2024.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+- [PixelAttack](https://jason1610.github.io/PixelAttack/) takes an image, pixelizes it and makes the pixels fight each other
+- [NeoHtop](https://github.com/Abdenasser/neohtop) is a modern, process monitoring tool that combines the power of terminal-based system monitors with a clean, desktop-native UI
+- [WebVM](https://github.com/leaningtech/webvm) is a Linux virtual machine that runs in your browser
+- [Sveltris](https://svelte.dev/playground/8594eea15c6244908c3efad284fe2aa4?version=5.2.1) is Tetris built entirely in the Svelte REPL
+
+### Learning Resources
+
+_Featuring Svelte Contributors and Ambassadors_
+
+- [Svelte 🧡](https://bsky.app/profile/ricciuti.me/feed/svelte-feed) is a Bluesky feed to keep up to date on Svelte news in that application
```
Comment (paoloricciuti) [inline apps/svelte.dev/content/blog/2024-12-01-whats-new-in-svelte-december-2024.md:54]:
```suggestion
- [Svelte 🧡](https://bsky.app/profile/paolo.ricciuti.me/feed/svelte-feed) is a Bluesky feed to keep up to date on Svelte news in that application
```

It should work the same but I changed my handle so better to have the right one

File: apps/svelte.dev/content/blog/2024-12-01-whats-new-in-svelte-december-2024.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+- [SvelteKit Streaming: The Complete Guide](https://khromov.se/sveltekit-streaming-the-complete-guide/) by Stanislav Khromov
+- [Remote Workshop: Svelte 5 & Runes](https://ti.to/mainmatter/svelte-5-runes-feb-2025) is taking place February 6-7th, 2025 and taught by Paolo Ricciuti
+- [Svelte 5: In Action | Build A Tasks App](https://www.youtube.com/watch?v=uSWMvDPpG0k) by Syntax
+- [Svelte Society London](https://www.youtube.com/watch?v=mxM208nCvHc) - featuring Paolo with his talk, *The joy of side projects ✨*
```
Comment (paoloricciuti) [inline apps/svelte.dev/content/blog/2024-12-01-whats-new-in-svelte-december-2024.md:58]:
```suggestion
- [Svelte Society London](https://www.youtube.com/watch?v=mxM208nCvHc) - featuring Paolo with his talk, *Herd your async tasks!*
```
Also in the same Livestream there's also the talk "Oh no SEO" from @spences10

File: apps/svelte.dev/content/blog/2024-12-01-whats-new-in-svelte-december-2024.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+*Everything Else*
+- [Svelte Multitone Image](https://stephane-vanraes.github.io/svelte-multitoneimage/) is a simple image renderer to apply multitone effects for Svelte
+- [number-flow](https://github.com/barvian/number-flow) is a component to transition, format, and localize numbers
+- [sveltednd](https://github.com/thisuxhq/SvelteDnD) is a lightweight, flexible drag and drop library for Svelte 5 applications
+- [ShadEditor](https://github.com/Tsuzat/ShadEditor) is a highly extensible text editor for svelte made with TipTap and ShadCN UI
+- [Tipex](https://www.npmjs.com/package/@friendofsvelte/tipex) is an advanced rich text editor based on Tiptap and Prosemirror
+- [svelte-bundle](https://github.com/uhteddy/svelte-bundle) is a tool for bundling Svelte components into a single HTML file
+- [svelte5-router](https://github.com/mateothegreat/svelte5-router) is an SPA router that allows you to divide & conquer your app with nested routers
+
```
Comment (paoloricciuti) [inline apps/svelte.dev/content/blog/2024-12-01-whats-new-in-svelte-december-2024.md:97]:
```suggestion
- [@sheepdog/svelte](https://github.com/main matter/sheepdog) is a library to manage async tasks and concurrency with ease.

```

**Review by paoloricciuti — COMMENTED** [review]
Left a couple of corrections and a suggestion for a library

File: apps/svelte.dev/content/blog/2024-12-01-whats-new-in-svelte-december-2024.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+*Everything Else*
+- [Svelte Multitone Image](https://stephane-vanraes.github.io/svelte-multitoneimage/) is a simple image renderer to apply multitone effects for Svelte
+- [number-flow](https://github.com/barvian/number-flow) is a component to transition, format, and localize numbers
+- [sveltednd](https://github.com/thisuxhq/SvelteDnD) is a lightweight, flexible drag and drop library for Svelte 5 applications
+- [ShadEditor](https://github.com/Tsuzat/ShadEditor) is a highly extensible text editor for svelte made with TipTap and ShadCN UI
+- [Tipex](https://www.npmjs.com/package/@friendofsvelte/tipex) is an advanced rich text editor based on Tiptap and Prosemirror
+- [svelte-bundle](https://github.com/uhteddy/svelte-bundle) is a tool for bundling Svelte components into a single HTML file
+- [svelte5-router](https://github.com/mateothegreat/svelte5-router) is an SPA router that allows you to divide & conquer your app with nested routers
+- [@sheepdog/svelte](https://github.com/main matter/sheepdog) is a library to manage async tasks and concurrency with ease.
```
Comment (paoloricciuti) [inline apps/svelte.dev/content/blog/2024-12-01-whats-new-in-svelte-december-2024.md:96]:
```suggestion
- [@sheepdog/svelte](https://github.com/mainmatter/sheepdog) is a library to manage async tasks and concurrency with ease.
```

Ugh noticed this in the newsletter

**Review by dummdidumm — APPROVED** [review]

**Pull request merged**